kyuubi-common/src/test/scala/org/apache/kyuubi/KyuubiSQLExceptionSuite.scala

```diff
@@ -35,7 +35,6 @@ class KyuubiSQLExceptionSuite extends KyuubiFunSuite {
 
     val e3 = KyuubiSQLException(e2.toTStatus)
     assert(e3.getMessage === e2.getMessage)
-    assert(e3.getStackTrace === e2.getStackTrace)
```

Comment:
![image](https://user-images.githubusercontent.com/6757692/115964858-01743f00-a559-11eb-9d40-c462c57de23d.png)
In jdk11, the stack trace is not absolute same.